scripts/deploy.sh

```
@@ -0,0 +1,179 @@
+        sudo nginx -s reload
+        log "Nginx switched to port $new_port"
+        return 0
+    else
+        log "Nginx configuration test failed"
+        return 1
+    fi
+}
+
+# 메인 배포 로직
+deploy() {
+    local active_port=$(get_active_port)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)